packages/diffhtml/lib/node/create.js

```diff
@@ -26,6 +26,16 @@ export default function createNode(vTree, doc = document) {
   // Will vary based on the properties of the VTree.
   let domNode = null;
 
+  CreateNodeHookCache.forEach(fn => {
+    // Invoke all the `createNodeHook` functions passing along the vTree as the
+    // only argument. These functions must return a valid DOM Node value.
+    domNode = fn(vTree);
+  });
+
+  if (domNode) {
+    return domNode;
+  }
```

Comment:
Notice this is in `node/create`. This is a function that takes in a `VTree` and returns a DOM Node. It would be useful to set this hook if you wanted to do custom DOM Node creation.